mix.exs

```diff
@@ -15,16 +15,31 @@ try do
   defmodule AntikytheraInstanceExample.Mixfile do
     use Mix.Project
 
+    versions =
+      File.read!(Path.join(__DIR__, ".tool-versions"))
+      |> String.split("\n", trim: true)
+      |> Map.new(fn line -> [n, v] = String.split(line, " ", trim: true); {n, v} end)
+    @elixir_version Map.fetch!(versions, "elixir")
+
+    # Strictly enforce Erlang/OTP and Elixir version
```

Comment:
Let's add more explanations on this code block (or code blocks starting on l.18):
```ex
# Here we strictly enforce Erlang/OTP version during evaluations of `mix.exs`.
# Elixir version is also checked using `:elixir` key in `project/0`.
# Although these version checks are not mandatory for antikythera instances,
# this way an antikythera instance can control Erlang/Elixir versions used in its gear projects.
```

Reply:
fixed in 1c6426f